src/Scene/NodeProcess.js

```diff
@@ -99,6 +100,7 @@ NodeProcess.prototype.subdivideNode = function subdivideNode(node, camera, param
                 // update Imagery wmts
                 for (const layer of colorLayers) {
                     if (layer.tileInsideLimit(child, layer)) {
+                        TiledImageTools.computeTileWMTSCoordinates(child, layer);
```

Comment:
Ok I see. WMTS is a protocol. You're speaking about our *TileMatrixSet*. Let's call that `computeTileMatrixSetCoordinates`. Same for the other occurrences.